src/init.cpp

```diff
@@ -661,6 +661,7 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-rpcbind=<addr>", _("Bind to given address to listen for JSON-RPC connections. Use [host]:port notation for IPv6. This option can be specified multiple times (default: bind to all interfaces)"));
     strUsage += HelpMessageOpt("-rpccookiefile=<loc>", _("Location of the auth cookie (default: data dir)"));
     strUsage += HelpMessageOpt("-rpcuser=<user>", _("Username for JSON-RPC connections"));
+    strUsage += HelpMessageOpt("-rpccorsdomain=<value>", _("Domain from which to accept cross origin requests (browser enforced)"));
```

Comment:
This should also be removed, as this is misleading to a user who will think that this is a supported option.

Taggin @sidhujag for visibility